consensus/pacemaker/module.go

```diff
@@ -60,6 +64,16 @@ type pacemaker struct {
 	logPrefix string
 }
 
+// Structure to handle delaying block preparation (reaping the block mempool)
+// by adding a delay before the next prepare request
```

Comment:
```suggestion
// by adding a delay before the next prepare request to prevent block creation
// from moving at the speed of the network (i.e. responsivness).
// See Tendermint, which is not responsive, for similar configurations; https://docs.tendermint.com/v0.33/tendermint-core/configuration.html
```